client/my-sites/customer-home/main.jsx

```diff
@@ -367,7 +373,8 @@ const connectHome = connect(
 			hasChecklistData,
 			isChecklistComplete,
 			isSiteEligible: isSiteEligibleForCustomerHome( state, siteId ),
-			isStaticHomePage: 'page' === getSiteOption( state, siteId, 'show_on_front' ),
+			isStaticHomePage,
+			staticHomePageId: isStaticHomePage ? getSiteFrontPage( state, siteId ) : -Infinity,
```

Comment:
Could we set the prop to optional and then use

```suggestion
			staticHomePageId: getSiteFrontPage( state, siteId ),
```

Does `getSiteFrontPage` return `null` when one isn't set?